### PR TITLE
Understanding

### PR DESCRIPTION
**Pull request by MrButak**
I set up a basic file structure for a node.js express project. I also have a working function that will take user input and insert into the database.

### CODE REVIEW

**Comment by MrButak** [comment]
@JohnnyCurran

File: public/javascripts/display-paragraph.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// this function should display paragraph from database to appropriate story on the DOM
+
+exports.displayParagraph = (paragraph) => {
```
Comment (JohnnyCurran) [inline public/javascripts/display-paragraph.js:3]: The display paragraph method here is doing 2 things: 
- retrieve paragraph from database (data layer)
- prepare paragraph for display (view layer)

I’d recommend splitting these up into two different methods: one for paragraph retrieval that returns the paragraph, and one that prepares the paragraph for the view. I’d place the database related functions in a folder like JavaScripts/data/paragraph.js 

I’m not super familiar with the templating engine, pug, but you could leave the prepare paragraph method in this file for now

File: public/javascripts/insert-paragraph.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+
+
+exports.insertParagraph = (paragraph) => {
```
Comment (JohnnyCurran) [inline public/javascripts/insert-paragraph.js:3]: I would rename this file to /javascripts/data/paragraph.js. This file can be where all of your database functions related to creating, updating, reading, and deleting paragraphs from the DB will live

**Review by JohnnyCurran — CHANGES_REQUESTED** [review]

File: public/javascripts/display-paragraph.js (hunk 1 above)
Comment (MrButak) [reply public/javascripts/display-paragraph.js:3]: Sounds good. I did end up doing this.

**Pull request merged**